### PR TITLE
[Bug] CrowdStrike Falcon - we are not passing incident type to the classifier into raw json

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/40369

## Description
Added incident type to each incident

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by daryakoval** [comment]
Test fails [here](https://app.circleci.com/pipelines/github/demisto/content/107512/workflows/f54447aa-ecf7-4550-86f5-76e5223d6115/jobs/411640).

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_2_20.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+#### Classifiers
+##### CrowdStrike Falcon Incident Classifier
+- Fixed a bug where `CrowdStrike Falcon Detection` and `CrowdStrike Falcon Incident` incident types were not set correctly.
```
Comment (ShahafBenYakir) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_2_20.md:4]:
```suggestion
- Fixed a bug where `CrowdStrike Falcon Detection` and `CrowdStrike Falcon Incident` incident types were not classified correctly.
```

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py [hunk 2]
```diff
@@ -2244,6 +2244,26 @@ def test_new_fetch(self, set_up_mocks, mocker, requests_mock):
         assert demisto.setLastRun.mock_calls[0][1][0] == {'first_behavior_detection_time': '2020-09-04T09:16:11Z',
                                                           'detection_offset': 0}
 
+    def test_fetch_incident_type(self, set_up_mocks, mocker):
+        """
+        Tests the adding of incident_type field to the context
+        Given:
+            Old getLastRun which holds `first_behavior_time` and `last_detection_id`
```
[the remark this answers is not in the document]
Comment (daryakoval) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_test.py:2251]: previous

**Review by dantavori — CHANGES_REQUESTED** [review]

File: Packs/CrowdStrikeFalcon/ReleaseNotes/1_2_20.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#### Classifiers
+##### CrowdStrike Falcon Incident Classifier
+- Fixed an issue where **CrowdStrike Falcon Detection** and **CrowdStrike Falcon Incident** incident types were not classified correctly.
+
+#### Integrations
+##### CrowdStrike Falcon
+- Fixed an issue in ***fetch-incidents*** command where the *incident_type* field was not returned to the raw incident data.
```
Comment (roysagi) [inline Packs/CrowdStrikeFalcon/ReleaseNotes/1_2_20.md:8]:
```suggestion
- Fixed an issue in the ***fetch-incidents*** command where the *incident_type* field was not in raw incident data.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Review by dantavori — APPROVED** [review]

**Pull request merged**